Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -703,6 +740,10 @@ extension ActivityStreamPanel: DataObserverDelegate {
             site = Site(url: pocketStories[index].url.absoluteString, title: pocketStories[index].title)
             telemetry.reportEvent(.Click, source: .Pocket, position: index)
             LeanPlumClient.shared.track(event: .openedPocketStory, withParameters: ["Source": "Activity Stream" as AnyObject])
+        case .pocketVideo:
+            site = Site(url: pocketVideoStories[index].url.absoluteString, title: pocketVideoStories[index].title)
+            telemetry.reportEvent(.Click, source: .Pocket, position: index)
+            LeanPlumClient.shared.track(event: .openedPocketStory, withParameters: ["Source": "Activity Stream" as AnyObject])
```

Comment:
Do we want to add some other params here to indicate to telemetry/LP that the Pocket story clicked is a "video"?